examples/05_local_development/README.md

```diff
@@ -0,0 +1,18 @@
+# Local Development
```

Comment:
Consider calling this section something broader, like "development_lifecycle" instead of "local development"... I think a lot of the concepts in this section are a lot broader than just local development